src/main/flight/pid.c

```diff
@@ -342,9 +344,16 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
 
         // -----calculate I component
         float ITerm = previousGyroIf[axis];
-        if (motorMixRange < 1.0f) {
-            // Only increase ITerm if motor output is not saturated
-            ITerm += Ki[axis] * errorRate * dT * dynKi * itermAccelerator;
+        const float ITermNew = ITerm + Ki[axis] * errorRate * dT * dynKi * itermAccelerator;
+        bool ITermSaturated;
+        if (isTricopter && axis == FD_YAW) {
+            ITermSaturated = errorRate > 75; // heuristic used by TriFlight
```

Comment:
I don't like magic numbers - IMO it should be either configurable parameter or explained in the comments.